src/index.html

```diff
@@ -15,8 +15,47 @@
       rel="stylesheet"
       href="./style.css"
     />
+    <link
+      rel="preconnect"
+      href="https://fonts.googleapis.com"
+    />
+    <link
+      rel="preconnect"
+      href="https://fonts.gstatic.com"
+    />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Roboto:ital,wght@0,100;0,300;0,400;0,500;0,700;0,900;1,100;1,300;1,400;1,500;1,700;1,900&display=swap"
+      rel="stylesheet"
+    />
   </head>
+
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+      <a href="#">
+        <img
+        class="logo"
+        src="/src/images/logo.png"
+        alt="Moyo logo"
+        />
+      </a>
+      <nav class="navbar">
+        <ul class="list">
+          <li><a href="#" class="is-active nav-link nav-link-1">APPLE</a></li>
+          <li><a href="#" class="nav-margin nav-link">SAMSUNG</a></li>
+          <li><a href="#" class="nav-margin nav-link">SMARTPHONES</a></li>
+          <li>
+            <a
+            data-qa="hover"
+            href="#"
+            class="nav-margin nav-link"
+            >LAPTOPS & COMPUTERS</a>
+          </li>
+          <li><a href="#" class="nav-margin nav-link">GADGETS</a></li>
+          <li><a href="#" class="nav-margin nav-link">TABLETS</a></li>
+          <li><a href="#" class="nav-margin nav-link">PHOTO</a></li>
+          <li><a href="#" class="nav-link nav-link-8">VIDEO</a></li>
```

Comment:
The text for the navigation links is hardcoded in uppercase. Consider using CSS to transform the text to uppercase instead of hardcoding it in HTML. This can be achieved using the `text-transform: uppercase;` CSS property.